snap/s4a/objects.js

```diff
@@ -37,7 +37,8 @@ SpriteMorph.prototype.initArduinoBlocks = function () {
         only: SpriteMorph,
         type: 'command',
         category: 'arduino',
-        spec: 'connect arduino at %s'
+        spec: 'connect arduino at %s',
+		translatable: true
```

Comment:
`translatable` here refers to translatable to C in the Arduino translation mode. This block is not translatable into C, as the resulting code runs inside the board, so we don't need to connect to it :)

I agree though, it should be called something else... transpilable?